### PR TITLE
feat: add support for disputes

### PR DESCRIPTION
**Pull request by kevcodez**
## What kind of change does this PR introduce?

Adds support for disputes

### CODE REVIEW

File: src/lib/sync.ts [hunk 1]
```diff
@@ -72,6 +75,9 @@ export async function syncBackfill(params?: SyncBackfillParams): Promise<SyncBac
     case 'payment_method':
       paymentMethods = await syncPaymentMethods(created)
       break
+    case 'dispute':
+      disputes = await syncDisputes(created)
```
Comment (phamhieu) [inline src/lib/sync.ts:79]: should be added to the return object

File: src/lib/disputes.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import { getConfig } from '../utils/config'
+import { stripe } from '../utils/StripeClientManager'
+import { cleanseArrayField, constructUpsertSql } from '../utils/helpers'
+import { upsertCharge } from './charges'
+import { disputeSchema } from '../schemas/dispute'
+
+const config = getConfig()
+
+export const upsertDispute = async (charge: Stripe.Dispute): Promise<Stripe.Dispute[]> => {
+  // Backfill customer if it doesn't already exist
```
Comment (phamhieu) [inline src/lib/disputes.ts:13]: typo

File: src/lib/sync.ts (hunk 1 above)
Comment (kevcodez) [reply src/lib/sync.ts:79]: Adjusted

File: src/lib/disputes.ts (hunk 2 above)
Comment (kevcodez) [reply src/lib/disputes.ts]: Adjusted

**Review by phamhieu — APPROVED** [review]

**Pull request merged**